conans/client/tools/scm.py

```diff
@@ -220,3 +222,20 @@ def is_pristine(self):
     def get_revision(self):
         return self.run("info --show-item revision").strip()
 
+    def get_last_changed_revision(self):
+        return self.run("info --show-item last-changed-revision").strip()
```

Comment:
Here we will ask for the revision to the _root_folder_. Need to merge this (https://github.com/conan-io/conan/pull/3479) first.

Reply:
Already merged. We should implemente and use the `get_root_folder` for SVN.

Reply:
`get_root_folder` is available. But for me no sense to put into `get_last_changed_revision`.

Reply:
I think that we want to know the `last_changed_revision` of the checked-out project (I know we can have several projects in the same SVN), not the last one in the current folder (we can instantiate SVN in a nested folder). When I say `root` I refer to the root of the project, not the root of all the SVN repository.

Reply:
Added as default option the working copy root folder to the `last_changed_revision`